tools/pythonpkg/src/python_objects.cpp

```diff
@@ -403,9 +403,9 @@ py::object PythonObject::FromValue(const Value &val, const LogicalType &type) {
 	}
 	case LogicalTypeId::INTERVAL: {
 		auto interval_value = val.GetValueUnsafe<interval_t>();
-		uint64_t days = duckdb::Interval::DAYS_PER_MONTH * interval_value.months + interval_value.days;
-		return py::cast<py::object>(import_cache.datetime().timedelta()(
-		    py::arg("days") = days, py::arg("microseconds") = interval_value.micros));
+		return py::cast<py::object>(import_cache.pandas().DateOffset()(
```

Comment:
This change seems unrelated to the Arrow change, no? Could we perhaps revert this change in this PR and potentially address this later?

Reply:
Hi @Mytherin @pdet ,

I reverted code and updated tests. Please review it and give me any feedback

Thanks